packages/xo-server/src/api/vif.js

```diff
@@ -58,13 +58,14 @@ connect.resolve = {
 // -------------------------------------------------------------------
 
 export async function set({
-  vif,
-  network,
-  mac,
   allowedIpv4Addresses,
   allowedIpv6Addresses,
   attached,
+  mac,
+  network,
   rateLimit,
+  txChecksumming,
+  vif,
```

Comment:
IMHO, keep it above.

Reply:
Why?

Reply:
We set the VIF here, so it's better to keep it above